tests/functional/cli/test_cli_repository.py

```diff
@@ -45,6 +49,49 @@ def test_list_all_commits(gitlab_cli, project):
     assert len(ret_all.stdout) > len(ret.stdout)
 
 
+def test_commit_merge_requests(gitlab_cli, project, merge_request, wait_for_sidekiq):
+    """This tests the `project-commit merge-requests` command and also tests
+    that we can print the result using the `json` formatter"""
+    # create and then merge a merge-request
+    source_branch = "test_commit_merge_requests"
+    mr = merge_request(source_branch=source_branch)
```

Comment:
I didn't see that variable used anywhere- maybe we don't need it?

```suggestion
    mr = merge_request(source_branch="test_commit_merge_requests")
```

Reply:
Done.